website/project/model.py

```diff
@@ -75,16 +74,9 @@ def has_anonymous_link(node, auth):
     :param str link: any view-only link in the current url
     :return bool anonymous: Whether the node is anonymous to the user or not
     """
-    view_only_link = auth.private_key or request.args.get('view_only', '').strip('/')
```

Comment:
I mean, I guess we could ask @chennan47. :-)

@chennan47 Do you remember why you added `.strip('/')` [here](https://github.com/CenterForOpenScience/osf.io/commit/0de958fe85d12816b8e6594c3ec16bfaf6fc8726#diff-033d4b3df3102b7d1aace074938585e9R470)? It seems to me to be at best unnecessary (when would a private link key be wrapped in backslashes? every read I've seen of it is from the querystring, not a path part) and at worst buggy (are we sure that '/' is and will never be a valid character in a private link key?).